question1.java

```diff
@@ -0,0 +1,136 @@
+package question1;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+/**
+ * Custom Scanner which reads lines from a file and parses the information
+ * according to what the user is expecting.
+ * 
+ * @author Alexa
+ *
+ */
+public class MyScanner {

```

Comment:
I used it for several assignments as it uses StringTokenizer and is much faster, but since the input is small, only two Strings, I might as well use the Standard scanner. I changed it now.